backend/tracim_backend/lib/core/event.py

```diff
@@ -235,6 +241,15 @@ def _has_just_been_deleted(self, obj: typing.Union[User, Workspace, ContentRevis
             return history.has_changes()
         return False
 
+    def _has_just_been_undeleted(
+        self, obj: typing.Union[User, Workspace, ContentRevisionRO]
+    ) -> bool:
+        """Check that an object has been undeleted since it has been queried from database."""
```

Comment:
```suggestion
        """Check that an object has been undeleted since queried from database."""
```
In the current formulation, `since` may mean `puisque`, this suggestion is an attempt to remove the ambiguity.